src/Public/Pages.ps1

```diff
@@ -854,11 +863,33 @@ function ConvertTo-PodeWebPage
             New-PodeWebTab -Name $name -Layouts $form
         })
 
-        $group = [string]::Empty
-        if ($GroupVerbs) {
-            $group = $cmdInfo.Verb
-            if ([string]::IsNullOrWhiteSpace($group)) {
-                $group = '_'
+        switch ($PSCmdlet.ParameterSetName) {
+            'NoGrouping' {
+                $group = [string]::Empty
+                break
+            }
+
+            'GroupByVerbs' {
+                $group = [string]::Empty
+                $group = $cmdInfo.Verb
+                if ([string]::IsNullOrWhiteSpace($group)) {
+                    $group = '_'
+                }
+                break
+            }
+
+            'GroupByCustomKey' {
+                $group = $Group -replace "\.",""
```

Comment:
I was wondering why you were replacing the dots, but I'm guessing it's because it stops the groups expanding? This looks like the same issue as with spaces.

This line here: https://github.com/Badgerati/Pode.Web/blob/7db656cb48fad17623b832573ff038f8c475ee22/src/Private/Helpers.ps1#L347

Changing `\s` to `[\s\.]` should fix this issue, and mean we can remove these two `replace '\.'` parts here :)